ui/app/mirrors/create/cdc/cdc.tsx

```diff
@@ -46,12 +46,37 @@ export default function CDCConfigForm({
   };
 
   const normalSettings = useMemo(() => {
-    return settings.filter((setting) => setting.advanced != true);
-  }, [settings]);
+    return settings!
+      .map((setting) => {
+        if (
+          IsQueuePeer(mirrorConfig.destination?.type) &&
+          setting.label === 'Sync Interval (Seconds)'
```

Comment:
rather than hardcoding it here, spec could be `advanced: 'queue'` but maybe makes types annoying